src/phoenix/server/main.py

```diff
@@ -120,9 +129,15 @@ def _get_pid_file() -> Path:
             ),
         ):
             traces.put(span)
+    umap_params_list = args.umap_params.split(",")
+    umap_params = UMAPParameters(
+        float(umap_params_list[0]), int(umap_params_list[1]), int(umap_params_list[2])
```

Comment:
for clarity, let's make these keyword arguments (instead of positional)

Reply:
Ah, yes, very good point